src/deadline_test_fixtures/deadline/worker.py

```diff
@@ -434,7 +436,7 @@ def _start_worker_agent(self) -> None:
                 [
                     "echo 'Running Get-Process to check if the agent is running'",
                     'for($i=1; $i -le 30 -and "" -ne $err ; $i++){sleep $i; Get-Process pythonservice -ErrorVariable err}',
-                    "IF(Get-Process pythonservice){echo 'service is running'}ELSE{exit 1}",
+                    "IF(Get-Process pythonservice){echo '+++SERVICE IS RUNNING+++'}ELSE{echo '+++SERVICE NOT RUNNING+++'; Get-Content -Encoding utf8 C:\ProgramData\Amazon\Deadline\Logs\worker-agent-bootstrap.log,C:\ProgramData\Amazon\Deadline\Logs\worker-agent.log; exit 1}",
```

Comment:
I found failures in this script to be challenging to debug if the agent doesn't start. The reason was typically something discoverable in the agent logs, so I've added outputting the logs on error.

Reply:
👏 👏 👏 This is so handy